src/Type/ObjectType/TermObject.php

```diff
@@ -56,7 +56,8 @@ public static function register_taxonomy_object_type( WP_Taxonomy $tax_object )
 							$url = $term->link;
 							if ( ! empty( $url ) ) {
 								$parsed = wp_parse_url( $url );
```

Comment:
TBH we should probably stop using `wp_parse_url()` to get a `uri` throughout the codebase. Much better to use [`wp_make_relative_uri()`](https://developer.wordpress.org/reference/functions/wp_make_link_relative/) and strip the possible localhost port. Good candidate for a Utils function.